featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -698,6 +700,8 @@ def test_boolean_multiply(boolean_mult_es):
     es = boolean_mult_es
     to_test = [
         ("numeric", "numeric"),
+        ("numeric", "bool"),
```

Comment:
@rwedge These were removed in the PR with changes to `MultiplyNumeric`, so had to add them back in now.